db/migrate/20150630025128_create_tenant.rb

```diff
@@ -7,8 +7,14 @@ def change
       t.string :appliance_name
 
       t.string :login_text
-      t.attachment :logo
-      t.attachment :login_logo
+      t.string   :logo_file_name
+      t.string   :logo_content_type
+      t.integer  :logo_file_size
+      t.datetime :logo_updated_at
+      t.string   :login_logo_file_name
+      t.string   :login_logo_content_type
+      t.integer  :login_logo_file_size
+      t.datetime :login_logo_updated_at
```

Comment:
problem: helper `attachment` is not available.
solution: manually create the fields that were created by the attachment helper.
result: `schema.rb` (and database) should be identical.
check: the `schema.yaml` should freak if these values are different.

In the future, I'd like to remove these values.